collection/lens.rkt

```diff
@@ -39,6 +67,21 @@
   (make-lens (λ (target) (nth target index))
              (λ (target value) (set-nth target index value))))
 
+(define (mapper-lens lens)
```

Comment:
Otherwise there would be confusion about whether `(map-lens ...)` is a lens itself, or if it's a function that operates on lenses and lists.
Making the names of lenses sound like nouns removes that potential ambiguity. (at least that's how I think)